doc/expo.md

```diff
@@ -25,7 +25,7 @@ To start debugging in Expo follow these steps:
 
 ## Configuring Expo
 
-The extension supports running through Exponent not just the applications created with Expo but even pure React Native applications (in that case you may need to add expo package to devDependencies in order to make it work with Expo: `npm install expo --save-dev`). In either cases it uses `app.json` configuration file in the root of the project.
+The extension supports running through Exponent not just the applications created with Expo but even pure React Native applications (in that case you may need to add `expo` package to `node_modules` in order to make it work with Expo: `npm install expo --no-save`). In either cases it uses `app.json` configuration file in the root of the project.
```

Comment:
```suggestion
The extension supports running through Exponent not just the applications created with Expo but even pure React Native applications (in that case you may need to add `expo` package to `node_modules` in order to make it work with Expo: `npm install expo --no-save`. Do not add it to `package.json`, because otherwise extension won't be able to distinguish your React Native project from expo ones!). In either cases it uses `app.json` configuration file in the root of the project.
```